app-modules/service-management/tests/Filament/ServiceMonitoring/Pages/CreateServiceMonitoringTest.php

```diff
@@ -139,3 +139,39 @@
     expect($serviceMonitoringTarget->teams())->exists()->toBeTrue();
     expect($serviceMonitoringTarget->users())->exists()->toBeTrue();
 });
+
+test('create service monitoring with validate the domain url', function () {
+    asSuperAdmin();
+
+    $validUrls = [
+        'http://example.com',
+        'http://192.168.0.1',
+        'http://192.168.0.1:8000',
```

Comment:
An IP address typically does not have the protocol defined. Which your regex rule allows, the `url` rule was preventing it though.. Remove them here.